lib/string_utils.rb

```diff
@@ -8,4 +8,8 @@ def split_string(string, limit:)
     end
     [words1.join(' '), (words2.join(' ')[0...limit])]
   end
+
+  def redact(string, first: 0, last: 0, redaction: '...')
```

Comment:
```suggestion
  def redact(string, first: 0, last: 0, redaction: '…')
```

Can we use the ellipsis character everywhere, rather than three dots.